savoirfairelinux/num2words
# fix an error with Vietnamese which can't use in command line

### Changes proposed in this pull request:

* Add support for Vietnamese in the command line

### Status

- [x ] READY
- [] HOLD
- [] WIP (Work-In-Progress)

### How to verify this change
At the command line, run `num2words {a_number_you want} -l vi ` or at code you can run `num2words("{a_number_you_want}", lang="vi")`. For example, `num2words("21", lang="vi")` should return "hai mươi mốt"
